pkg/api/pluginproxy/ds_proxy_test.go

```diff
@@ -583,11 +584,62 @@ func TestNewDataSourceProxy_ProtocolLessURL(t *testing.T) {
 	}
 	cfg := setting.Cfg{}
 	plugin := plugins.DataSourcePlugin{}
+
 	_, err := NewDataSourceProxy(&ds, &plugin, &ctx, "api/method", &cfg)
 
 	require.NoError(t, err)
 }
 
+// Test wth MSSQL type data sources.
+func TestNewDataSourceProxy_MSSQL(t *testing.T) {
```

Comment:
Testing the proxy with MSSQL doesn't make sense.

Reply:
@marefr Why is that a problem? The proxy supports data sources of MSSQL type, same as the handlers for adding and updating data sources. The benefit of testing from this level is that you're testing fewer implementation details, and you can see that the chain of code is working.

Reply:
@marefr OK after our conversation I understand what you mean. I'll try to look into it.